src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -82,16 +84,15 @@
         "thread-network-diagnostics-cluster.xml",
         "time-format-localization-cluster.xml",
         "time-synchronization-cluster.xml",
-        "user-label-cluster.xml",
-        "unit-localization-cluster.xml",
-        "wake-on-lan-cluster.xml",
-        "wifi-network-diagnostics-cluster.xml",
-        "window-covering.xml",
-        "matter-devices.xml",
         "types/color-control.xml",
         "types/door-lock.xml",
         "types/occupancy-sensing.xml",
-        "types/thermostat-user-interface-configuration.xml"
+        "types/thermostat-user-interface-configuration.xml",
```

Comment:
And here, what I would say we should do is, in order:

1. Clusters in alphabetical order
2. matter-devices
3. types/* in alphabetical order

Reply:
Reverted the sorting part of the PR